include/libtorrent/session_settings.hpp

```diff
@@ -229,6 +229,9 @@ namespace libtorrent
 		// time to wait until a new retry takes place
 		int urlseed_wait_retry;
 		
+		// if unavailable, retry in these minutes
+		int web_seed_name_lookup_retry_minutes;
```

Comment:
unfortunately, adding this breaks the ABI. You can add a new setting at the end to RC_1_1 though.